tox.ini

```diff
@@ -37,6 +37,8 @@ norecursedirs = .* docs requirements
 deps =
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r{toxinidir}/requirements/test.txt
 commands =
     py.test {posargs}
```

Comment:
To output any applicable deprecation warnings, this needs to be updated to `python -Wd -m pytest {posargs}`.